### PR TITLE
Fix tab switch

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes not able to connect after removing firmware cloud option when user was on a tab previous using that option.

To reproduce. 

- Enable remember last tab in options tab
- Flash firmware with servo support
- Click servo tab
- Flash firmware without servo support
- Press connect

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]

File: src/js/gui.js
Comment (chmelevskij) [inline src/js/gui.js:358]: Would something liek this work?
```suggestion
        const tab = result.rememberLastTab && result.lastTab && this.allowedTabs.includes(result.lastTab.substring(4)) ? result.lastTab : 'tab_setup';
```

Comment (haslinghuis) [reply src/js/gui.js]: :grin: Thanks :+1:

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
